packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts

```diff
@@ -67,11 +67,14 @@ export default {
       removeButton: 'Fjern',
       doneButton: 'Ferdig',
       cancelButton: 'Avbryt',
+      clearButton: 'Tøm',
       errorInContainer: 'Feilene ovenfor må rettes.',
     },
     IteratePushContainer: {
       createButton: 'Legg til',
       itemsLimitReached: 'Du har nådd grensen på: {limit}',
+      requireCommitText:
+        'Du har endringer som enten må legges til eller tømmes.',
```

Comment:
```suggestion
        'Du har endringer som enten må legges til eller slettes.',
```

Unsure, but perhaps `slettes` is a better word than `tømmes`?

Reply:
Right now we call the "clear" button text for "Tøm". This button is shown when this text appears. So I would like to have a relation if possible.

Reply:
I see the button says "tøm".

Do we have an other component/example of the same functionality?

Like clearing data in date picker, or so?

I'm just interested in that we chose the same wording as in other places.